webapp/index.html

```diff
@@ -12,14 +12,23 @@
 		data-sap-ui-libs="sap.m"
 		data-sap-ui-bindingSyntax="complex"
 		data-sap-ui-compatVersion="edge"
-		data-sap-ui-preload="async"
```

Comment:
Ohh yeah, we will readd this flag. The removal makes only sense for the internal version.